File: src/2639.find-the-width-of-columns-of-a-grid.2675/solution.ts
function findColumnWidth (grid: number[][]): number[] {
    const M = grid.length;
    const N = grid[0].length;
    const result = new Array<number>(N);
    for (let j = 0; j < N; j++) {
        let item = 0;
        for (let i = 0; i < M; i++) {
            item = Math.max(item, String(grid[i][j]).length);
        }
        result[j] = item;
    }
    return result;
}

File: src/2639.find-the-width-of-columns-of-a-grid.2675/solution.java
class Solution {
    public int[] findColumnWidth(int[][] grid) {
        int M = grid.length;
        int N = grid[0].length;
        int[] result = new int[N];
        for (int[] row : grid) {
            for (int j = 0; j < N; j++) {
                result[j] = Math.max(result[j], String.valueOf(row[j]).length());
            }
        }
        return result;
    }
}